front-end/src/app/app.component.html

```diff
@@ -1 +1 @@
-<h1>{{ title }}</h1>
\ No newline at end of file
+<h1>{{ title.test }}</h1>
```

Comment:
if title is null there will be an error here
to be sure you have no error you can do with typescript
`<h1>{{ title?.test }}</h1>`
it will check if title is null or not

Reply:
still not good, take the code I wrote to you here